### DISCUSSION TITLE
How to Use Whisper for Feature representation？

### DISCUSSION THREAD

**Discussion by YiYang-github** [openai/whisper#1837] [Q&A]
I've noticed that Whisper is capable of speech translation and language identification, but it appears to lack the ability to directly convert audio input into a structured feature representation, like a 1024-dimensional vector. For my project, I am looking to process a dataset of Chinese audio clips, each containing a single word, and I would like to use Whisper to perform initial feature extraction. Could you provide guidance on how this might be achieved, or suggest alternative methods if Whisper isn't suited for this type of feature extraction?

**Comment by EtienneAb3d** [comment]
What you need is here:
https://github.com/facebookresearch/LASER
;-)
PS: works on text, not sound, thus after Whisper use. Knowing that Whisper is probably not really accurate on single-word audio.

**Reply by YiYang-github** [reply]
Thank you for the LASER recommendation. 
However, I'm looking for audio signal representation like Wav2Vec, capturing features like timbre and intonation, not just text transcriptions. Still, I appreciate your suggestion!

**Comment by perpetual-pj** [comment]
请问找到办法了吗

**Reply by superFilicos** [reply]
找到了
